src/pages/NotFound/index.jsx

```diff
@@ -13,7 +13,11 @@ const NotFound = () => {
         code="404"
         heading="Page Not Found 🤔"
         paragraph="The page you are looking for does not exist."
-        button={<Button onClick={handleRoutingHome}>{"Go to Home"}</Button>}
+        button={
+          <Button onClick={handleRoutingHome}>
+            <p>Go to Home</p>
+          </Button>
```

Comment:
Button component의 child로 p 태그가 아닌 "Go to Home" 만 전달해도 될 것 같습니다!
```suggestion
          <Button onClick={handleRoutingHome}>Go to Home</Button>
```

Reply:
Button 컴포넌트의 `children` propTypes 를 `element` 으로 해두어서, string 값만 들어가면 런타임에서 에러가 나더라구요!

버튼에 p 태그 엘리먼트로 지금처럼 넣는다.
string 으로 넣는다. -> Button 컴포넌트는 무조건 문자열만 버튼으로 만들 수 있음
Button.propTypes = {
  onClick: PropTypes.func.isRequired,
  children: PropTypes.element.isRequired,
};
1 과 2 중에 선택 받겠습니다! @GreenteaHT @shkimjune

Reply:
개인적으로는 Button children에 element가 들어갈 일이 없을 것 같아서 2번 선택지가 좋을 것 같습니다!